compose-recipes/mariadb/faster_innodb.cnf

```diff
@@ -0,0 +1,8 @@
+[mysqld]
+innodb_buffer_pool_size = 4G
+innodb_log_buffer_size = 256M
+innodb_log_file_size = 2G
+innodb_log_files_in_group = 1
```

Comment:
the innodb configuration here differs from the `mysql` configuration because while `innodb_redo_log_capacity` replaced `innodb_log_file_size` and `innodb_log_files_in_group` for mysql, that is not the case for `mariadb`; `mariadb` makes use of the latter two configs still.

`innodb_redo_log_capacity` = `innodb_log_file_size * innodb_log_files_in_group`, so i chose go with one log group and match the capacity and file size.